frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -55,6 +60,36 @@ export const getTaskOverviewData = (task: Task, editable: boolean) => {
     },
   ];
 
+  let editedAtText: string;
+  if (!task.updatedAt) {
+    editedAtText = '-';
+  } else {
+    const updatedDate = new Date(task.updatedAt);
+    let datePart: string;
+    if (isYesterday(updatedDate))
+      datePart = `${i18n.t('Yesterday at <time>', {
+        time: updatedDate.toLocaleTimeString([], {
+          hour: '2-digit',
+          minute: '2-digit',
+        }),
+      })}`;
```

Comment:
```suggestion
      datePart = i18n.t('Yesterday at <time>', {
        time: updatedDate.toLocaleTimeString([], {
          hour: '2-digit',
          minute: '2-digit',
        }),
      });
```
same for all `datePart`s

Reply:
Good catch, I forgot them after I switched from string format to just using i18n parameters